pkg/controller/account/credentials_rotator.go

```diff
@@ -25,9 +25,10 @@ func (r *ReconcileAccount) RotateCredentials(reqLogger logr.Logger, awsSetupClie
 
 	//var awsAssumedRoleClient awsclient.Client
 	var roleToAssume string
-
+	accountIAMID := account.Labels[fmt.Sprintf("%s", awsv1alpha1.IAMUserIDLabel)]
```

Comment:
Why the `fmt.Sprintf` here?  Is there a chance this comes back as not a string? Not blocking, just curious.

Reply:
@iamkirkbater This is how its done in the account controller, so I copied it